ksl_alert_backend/user/userController.js

```diff
@@ -187,6 +187,19 @@ const updateEmail = (req, res) => {
     .catch(error => res.status(500).json(error));
 };
 
+// delete a User's query
+const deleteQuery = (req, res) => {
+  const { userId, queryId } = req.body;
```

Comment:
@nikhilkamineni, looks good! For consistency, in a future commit, should we change `userId` to `id` to match other endpoints? This looks like good though and going to approve.